app/views/handlers/feedback.py

```diff
@@ -72,14 +79,17 @@ def handle_post(self):
         session_data.feedback_count += 1
 
         feedback_metadata = FeedbackMetadata(
-            session_data.feedback_count,
-            self._schema.form_type,
-            session_data.language_code,
-            self._schema.region_code,
+            session_data.case_id,
             session_data.tx_id,
         )
 
         feedback_message = FeedbackPayload(
+            self._questionnaire_store.metadata,
+            self._questionnaire_store.response_metadata,
+            session_data.case_id,
+            self._schema,
+            session_data.feedback_count,
+            session_data.language_code,
```

Comment:
If python does this positionally I think the arguments for feedback_count and language_code need to be swapped.